src/mscorlib/src/System/Globalization/CultureInfo.cs

```diff
@@ -1402,10 +1365,8 @@ public CultureTypes CultureTypes
             s_userDefaultCulture = null;
 
             RegionInfo.s_currentRegionInfo = null;
-#if !FEATURE_CORECLR // System.TimeZone does not exist in CoreCLR
-            TimeZone.ResetTimeZone();
```

Comment:
`System.TimeZone` was actually added back to corelib (#11169), but looks like removing the ifdef was missed as part of that. I think we want to keep this line. cc: @tarekgh